tools/docker/conf/Apache-Php/DockerFile-Apache-Php

```diff
@@ -0,0 +1,5 @@
+FROM webdevops/php-apache-dev
+COPY conf/Apache-Php/php.ini /opt/docker/etc/php/php.ini
+COPY conf/Apache-Php/.bash_aliases /root
+RUN apt-get update && \
+    apt-get install -y mysql-client
```

Comment:
The mysql client is already available in your `mysql` container, do you really need to install here as well?

The short answer should be no, as it is not used by the PHP / Apache server.
In all cases, FYI you should keep your layers as small as possible. When you run `apt update`, some files are generated, but they are probably not useful for the next next steps. So a cleanup could be nice here, i.e with `&& rm -rf /var/lib/apt/lists/*`

Reply:
mysql-client was needed to launch phpunit tests in CLI, it uses mysqldump.